tests/pyfunc/test_spark.py

```diff
@@ -21,6 +22,11 @@
 import tests
 from mlflow.types import Schema, ColSpec
 
+import sklearn.datasets as datasets
+from collections import namedtuple, OrderedDict
+import sklearn.neighbors as knn
```

Comment:
```suggestion
import sklearn.neighbors import KNeighborsClassifier
```

`sklearn.neighbors` is not k-nearest neighbors.